test/letters/00-unauthed.e2e.spec.js

```diff
@@ -0,0 +1,18 @@
+const E2eHelpers = require('../e2e/e2e-helpers');
+const LettersHelpers = require('../e2e/letters-helpers.js');
+const LoginHelpers = require('../e2e/login-helpers');
+
+module.exports = E2eHelpers.createE2eTest(
+  (client) => {
+    const token = LoginHelpers.getUserToken();
+
+    LettersHelpers.initApplicationMock(token);
+
+    // Test flow for unauthed and LOA1 users
+    LoginHelpers.testUnauthedUserFlow(client, '/letters');
```

Comment:
This is to test that the `RequiredLoginView` doesn't let us in, right?

I wonder if it would make more sense to bundle all these into a single test that pokes all the apps that require the user to be logged in. I think it'd save time in setup and teardown of a browser instance. Multiply that by however many "privileged" apps we have (minus one) and it might be worth our time.

What thinkest thou?

Reply:
I agree! I think that would be much more efficient. I'll create a ticket for that and address that with the whole FE team at the next sync.